src/settings/MappingProfiles/detailsSections/edit/InvoiceDetailSection/InvoiceAdjustments.js

```diff
@@ -272,6 +282,8 @@ export const InvoiceAdjustments = ({
               label={<FormattedMessage id={`${TRANSLATION_ID_PREFIX}.invoice.invoiceAdjustments.field.type`} />}
               name={INVOICE_ADJUSTMENTS_FIELDS_MAP.TYPE(index)}
               currency={currency}
+              onChangeToPercent={onChangeToPercent}
+              onChangeToAmount={onChangeToAmount}
```

Comment:
why `format()` `parse()` can't be used here? In such case, we wouldn't need changes in stripes-acq-components